demos/azure-resource-optimization-toolkit/README.md

```diff
@@ -19,6 +19,7 @@ languages:
 ![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/demos/azure-resource-optimization-toolkit/BestPracticeResult.svg)
 ![Cred Scan Check]( https://azurequickstartsservice.blob.core.windows.net/badges/demos/azure-resource-optimization-toolkit/CredScanResult.svg)
 
+![Bicep Version](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/demos/azure-resource-optimization-toolkit/CredScanResult/BicepVersion.svg)
```

Comment:
The URL needs to match the others.  Use the following -
![Bicep Version](https://azurequickstartsservice.blob.core.windows.net/badges/demos/azure-resource-optimization-toolkit/CredScanResult/BicepVersion.svg)